components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/login.jsp

```diff
@@ -398,7 +398,7 @@
         <div class="container-fluid">
             <p><%=AuthenticationEndpointUtil.i18n(resourceBundle, "wso2.identity.server")%> | &copy;
                 <script>document.write(new Date().getFullYear());</script>
-                <a href="http://wso2.com/" target="_blank"><i class="icon fw fw-wso2"></i>
+                <a href=""<%=AuthenticationEndpointUtil.i18n(resourceBundle, "business.homepage")%> target="_blank"><i class="icon fw fw-wso2"></i>
```

Comment:
<%=AuthenticationEndpointUtil.i18n(resourceBundle, "business.homepage")%>  needs to be inside quotation marks

Reply:
Thanks!  Fixed 75dddc0337